### PR TITLE
Update README.md - Added CodeCov

### PR DESCRIPTION
**Pull request by charanpreet0**
Added CodeCov Badge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # HappyCows/HappierCows
 
-[![codecov](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-6pm-HappyCows/branch/main/graph/badge.svg?token=F9AydNAnCV)](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-6pm-HappyCows)
+[![codecov](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-happycows/branch/master/graph/badge.svg?token=YedCF0lpyN)](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-happycows)
```
Comment (pconrad) [inline README.md:3]: Try `main` not `master`

**Review by pconrad — CHANGES_REQUESTED** [review]
`main` not `master`

**Review by peterbrede — APPROVED** [review]
Great badge

**Review by pconrad — APPROVED** [review]
LGTM

**Review by calvitronic — APPROVED** [review]
LGTM now that master is changed to main.

**Pull request merged**